test/evp_extra_test.c

```diff
@@ -1833,7 +1959,133 @@ static int test_decrypt_null_chunks(void)
 }
 #endif /* !defined(OPENSSL_NO_CHACHA) && !defined(OPENSSL_NO_POLY1305) */
 
-#if !defined(OPENSSL_NO_DH) && !defined(OPENSSL_NO_DEPRECATED_3_0)
+#ifndef OPENSSL_NO_DH
+/* Test combinations of private, public, missing and private + public key 
```

Comment:
Format should be the following for multi-liners.
/*
 * Test ...
 * ....
 */
Same applies to the function above..